client/src/components/Reviews.jsx

```diff
@@ -21,6 +21,30 @@ const Reviews = () => {
     getAllReviews();
   }, []);
 
+  const updateVotes = (e, val, reviewId) => {
+    let field = 'helpful';
+    if (e === 'voteUp') {
+      val += 1;
+    } else if (e === 'voteDown') {
+      val -= 1;
+    } else {
+      val += 1;
+      field = 'funny';
+    }
+
+    axios.patch(`http://localhost:4200/api/reviews/${id}`, {
```

Comment:
Just looked this up. I might have to update my put request to patch. Your way seems better. Thoughts?

Reply:
I used patch because to my understanding its better utilized to update a small portion of the data rather than a bunch of data